fix: tests

zze0s opened the pull request:
Fix tests

s0up4200 commented:
I can't push to this one, but could it really be just this?

```diff
diff --git a/util_windows.go b/util_windows.go
index d0fc4fc..4a5b7cd 100644
--- a/util_windows.go
+++ b/util_windows.go
@@ -30,6 +30,5 @@ func shellRun(line, dir string) (string, error) {
 }
 
 func isEscapeRune(r rune) bool {
-	// TODO: Implement escaping (via `) on Windows
-	return false
+	return r == '\\'
 }
```

zze0s commented:
> I can't push to this one, but could it really be just this?
> 
> ```diff
> diff --git a/util_windows.go b/util_windows.go
> index d0fc4fc..4a5b7cd 100644
> --- a/util_windows.go
> +++ b/util_windows.go
> @@ -30,6 +30,5 @@ func shellRun(line, dir string) (string, error) {
>  }
>  
>  func isEscapeRune(r rune) bool {
> -	// TODO: Implement escaping (via `) on Windows
> -	return false
> +	return r == '\\'
>  }
> ```

Fixed your privs.

I'm pretty sure the code needs to be updated and fixed instead of changing the test case, but not 100% 😄
